be/src/exprs/agg/java_udaf_function.h

```diff
@@ -125,26 +128,24 @@ class JavaUDAFAggregateFunction : public AggregateFunction {
         udf_ctxs->buffer =
                 std::make_unique<DirectByteBuffer>(udf_ctxs->buffer_data.data(), udf_ctxs->buffer_data.size());
         // chunk size
-        auto buffer_array = helper.create_object_array(udf_ctxs->buffer->handle(), chunk_size);
+        auto buffer_array = helper.create_object_array(udf_ctxs->buffer->handle(), batch_size);
+        LOCAL_REF_GUARD_ENV(env, buffer_array);
         jobject state_and_buffer[2] = {rets, buffer_array};
         helper.batch_update_state(ctx, ctx->impl()->udaf_ctxs()->handle.handle(),
                                   ctx->impl()->udaf_ctxs()->serialize->method.handle(), state_and_buffer, 2);
-        std::vector<Slice> slices(chunk_size);
```

Comment:
batch size, why just rename chunk size to batch size?

Reply:
Other interfaces (batch_update/merge/finalize) are all batch_size. I keep them consistent